packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -415,12 +415,12 @@ export class LitContracts {
     }
 
     this.log('Your Signer:', this.signer);
-    this.log('Your Provider:', this.provider.connection);
+    this.log('Your Provider:', this.provider?.connection!);
```

Comment:
should we move this log under the check for `this.provider` being defined.

Reply:
You mean wrap it inside a if/else to check if 'connection' exists then print it out? If so, I don't think so, cus I'd like to see it connection is undefined in different environments

Reply:
I see, if we want to log out when `this.provider` is undefined then we can keep  it.